### ISSUE TITLE
Uncaught (in promise) TypeError: t.data.sort is not a function

### ISSUE THREAD

**Issue by ArpitPy** [ArpitPy/whistleblower#2]
The component Feed.jsx in components folder of src contains data.sort to sort the posts from newest to oldest but it shows this error in console.

**Comment by Bibhavshah** [comment]
Can I go ahead and look for this error?

**Comment by ArpitPy** [comment]
Sure, assigned.

**Comment by Bibhavshah** [comment]
I think there is something wrong with your backend can you please check it once?

**Comment by ArpitPy** [comment]
Did you clone it and run on your local machine?
It works well on my laptop, fetches data from database, creates new users and posts, etc.
But when i host it on online after doing npm run build, it doesn't work.

**Comment by Bibhavshah** [comment]
Yeah, I did all of these and when I am running it on my machine the API is not working

**Comment by ArpitPy** [comment]
Oh...ok then, i will check it and let you know what i find!

**Comment by Saurav-Singh-Rauthan** [comment]
is this fixed?

**Comment by ArpitPy** [comment]
No it isn't

**Comment by Saurav-Singh-Rauthan** [comment]
Can I try?

**Comment by Bibhavshah** [comment]
The main error with this function is that it is not getting the desired input to perform the task which is written. That can also be because of the fetch request. so the code that I wrote simply checks whether the result we got from the fetch request is an array or not if not then it sets an empty array to the state. So, I think that is the error of the API which you are fetching.

**Comment by ArpitPy** [comment]
Yeah I think it is the API error, I will have to check it.

**Comment by Bibhavshah** [comment]
Hey, did you check the API, is it working fine? #ArpitPy

**Comment by ArpitPy** [comment]
Yes it is.
